### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN045.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#94] [General]
# https://blog.3place.cn/3place/lmyn/LMYN045.html

https://blog.3place.cn/3place/lmyn/LMYN045.html

**Comment by picklecai** [comment]
他同罗什曼那一起，
听着众友那美妙故事；
他们心里思考着它，
一整夜就这样消逝。 （1.44.3）

暗夜已经逝去，
我听到最值得听的故事。
具有大苦行功德的人呀！
这一夜好像转瞬即逝。
我心里反复地思考着，
你讲过的那全部事实。 （1.44.5）

罗摩是个听完故事反复建构的孩子。

**Comment by picklecai** [comment]
吠舍罗城

吠舍离（Vaishali）是古印度佛陀时代最大且最为繁盛的城市之一，坐落在根德格河畔，该城与佛陀时代的许多古城一样最终消亡，其确切地址的考定主要归功于中国僧人法显和玄奘的记载。
这里是佛陀最后一次正式集合僧众说法的地方。离开吠舍离前往涅槃地拘尸那迦。吠舍离百姓苦苦相留，佛陀留下钵作为纪念，这也是后来印度佛塔都建成覆钵形的原因。
这里除了阿育王柱和佛塔遗址外，还有吠舍离古塔遗迹，在当地称为一号塔，此遗址中挖掘出的舍利则被保存在巴特那博物馆。

**Comment by picklecai** [comment]
罗摩呀！古时候在圆满时〔358〕，

神话说，这个世界分为四“时”（yuga）。
这四个“时”是：一、圆满时（Kṛtayuga）；
二、三分时（Tretāyuga）；
三、二分时（Dvāyuga）；
四、争斗时（Kaliyuga）。

四时长短不一，圆满时长1728000年；三分时长1296000年；二分时长864000年；争斗时长432000年；四者加起来共有4320000年。有一个说法认为这就是一劫，与注24不同。
据说，前三时已过，我们现在正处于争斗时。争斗时开始于公元前3102年二月十七日至十八日的夜半时分。四时比较起来，时间越来越短，人的体质和道德也越来越坏。争斗时一结束，也就是劫末了，世界就要毁灭。
这种颂古非今的想法，许多国家古代都有过。九斤老太那种“一代不如一代”的想法，也属于这一类。

类似于分久必合合久必分，不过比合分二分法更精细。

**Comment by picklecai** [comment]
这些聪明的人
左思右想，他们这样想道：
’如果我们把那牛奶海搅动，
我们就会搅出不死之药。’ （1.44.16）

他们就这样想着想着，
把婆苏吉龙王〔360〕做成搅绳，
又把曼多罗山〔361〕做成搅棍，
神力无量的人把大海搅动。 （1.44.17）
首先搅出的是檀槃陀哩〔362〕，

檀槃陀哩，据说是神仙的医生，《生命吠陀》（Āyurveda）的作者。他后来转生为帝婆陀娑（Divodāsa），生于迦湿国，是印度医学学派的创始人。

一个医生，首先出世，真有道理。

**Comment by picklecai** [comment]
这些美妙秀丽的天女，
总数一共有十六亿〔363〕。
罗摩呀！她们还带着
数也数不清的女婢。 （1.44.19）

所有的神仙和檀那婆，
没有一个想娶她们。
因为没有人把她们娶，
她们就成了公共的女人〔364〕。 （1.44.20）

公共的女人。
这是妓女起源，乾达婆还不算。

**Comment by picklecai** [comment]
底提的儿子：阿修罗，没有接受美女。双方战斗中输了，被杀光了。
阿底提（因陀罗）的儿子们：修罗。接受了美女。双方战斗中打赢了。因此天帝释统治了天宫。

所以，天帝释来自因陀罗呀。

**Reply by picklecai** [reply]
原来天帝释就是因陀罗。

**Comment by picklecai** [comment]
他们就这样想着想着，
把婆苏吉龙王〔360〕做成搅绳，
又把曼多罗山〔361〕做成搅棍，
神力无量的人把大海搅动。 （1.44.17）

不是一根棍子就够了吗？绳子用在哪里？

**Comment by picklecai** [comment]
如果我们把那牛奶海搅动，
我们就会搅出不死之药。

为什么他们觉得搅动牛奶海，可以搅出不死药呢？
感觉像我小时候，自己定义某个动作，会产生某个结果。于是就去做。然后无论得到什么，都定义为这个结果。
所以这就是个儿童游戏过程。

**Reply by picklecai** [reply]
从下文看，是觉得牛奶海里啥都有。只要搅动，就会浮现。
